Reducers/ScheduleReducer.js

```diff
@@ -0,0 +1,93 @@
+import {
+    FILTER_PERFORMANCES,
+    LOAD_PERFORMANCES_BEGIN,
+    LOAD_PERFORMANCES_SUCCESS,
+    LOAD_PERFORMANCES_END
+} from '../Actions/ScheduleActions/ScheduleActionTypes';
+
+let currentDate = new Date();
+let dateAfterWeek = new Date(currentDate.getFullYear(), currentDate.getMonth(), currentDate.getDate() + 7);
+
+const initialState = {
+    performances: [
```

Comment:
backend has possibility to filter by start/enddate
What do you think about backend filtering
It will simplify frontend